### ISSUE TITLE
Do `try` blocks push a label on the label stack?

### ISSUE THREAD

**Issue by binji** [WebAssembly/exception-handling#7]
I assume that a `try` block introduces a new label, but I couldn't find it mentioned in the current overview.

e.g.
```
block
  try
    br 0 ;; does this go to A or B?
  else
    nop
  end
  ;; A
end
;; B
```

**Comment by KarlSchimpf** [comment]
This is a good point. Try blocks should be like other constructs and have a label. I will write a PR to fix this.

**Comment by KarlSchimpf** [comment]
I believe that this has been corrected with PR #12. Closing. Feel free to reopen if you disagree.
